### PR TITLE
Change requests to resume interrupted download of over-the-air (OTA)

### PR DESCRIPTION
**Pull request by madhawap**
1. Changes to /cache, /cache/ota directory clearance logic
2. Handled the connection failures, timeouts when downloading the update package (Additionally included download over ethernet)
3. Minor modifications to the logic of the firmware download

### CODE REVIEW

File: system-service/app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <resources>
-    <string name="app_name">WSO2 EMM System Service</string>
-    <string name="shared_pref_package">org.wso2.emm.system.service</string>
+    <string name="app_name">WSO2 IoT System Service</string>
```
Comment (charithag) [inline system-service/app/src/main/res/values/strings.xml]: As this PR is made for agent-emm-2.1.0, This should be EMM

File: system-service/app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+    <string name="shared_pref_package">org.wso2.iot.system.service</string>
```
Comment (charithag) [inline system-service/app/src/main/res/values/strings.xml]: As this PR is made for agent-emm-2.1.0, This should be org.wso2.emm.system.service

Comment (madhawap) [reply system-service/app/src/main/res/values/strings.xml]: Made the relevant change

File: system-service/app/src/main/res/values/strings.xml (hunk 1 above)
Comment (madhawap) [reply system-service/app/src/main/res/values/strings.xml]: Made the relevant change

**Pull request merged**